pkg/ivd/ivd_protected_entity.go

```diff
@@ -293,6 +317,13 @@ func (this IVDProtectedEntity) Snapshot(ctx context.Context, params map[string]m
 	retryInterval := time.Second
 	retryCount := 0
 	retrieveSnapDetailsErr := 0
+	this.logger.Infof("Checking vCenter connections...")
```

Comment:
Can we move the VC conn check inside the ipetm.vslmManager for all of FCD APIs?

Reply:
vslm manager is derived from the vcenter connection, hence it can't be moved in.

Reply:
You could implement wrapper calls in ipetm